apps/communications/contacts/matching_contacts.html

```diff
@@ -65,8 +65,12 @@ <h1 id="title" data-l10n-id="mergeDuplicateTitle"></h1>
               <aside class="pack-end">
                 <img data-src="#thumb#">
               </aside>
-              <p>#displayName#</p>
-              <bdi class="match-main-reason">#mainReason#</bdi>
+              <p>
```

Comment:
There is alternative solution - use ```bdi``` instead of ```p```, but then we should duplicate BB styles to make it look in the same way + we'll need ellipsis fix for long names/numbers anyway. Honestly I'd prefer ```p -> bdi``` solution, but tell me what do you think?